actionpack/lib/action_controller/renderer.rb

```diff
@@ -20,7 +20,7 @@ module ActionController
   # #render method allows you to use any options as when rendering in controller.
   # For example,
   #
-  #   FooController.render :action, locals: { ... }, assigns: { ... }
+  #   FooController.render :index, locals: { ... }, assigns: { ... }
```

Comment:
I thought that adding an example of action like `index` would be better instead of just word `:action` :)